.azure-pipelines/azure-pipelines-build-ubuntu-2204.yml

```diff
@@ -25,16 +25,16 @@ stages:
 
   jobs:
   - job:
-    displayName: "amd64/ubuntu-20.04"
+    displayName: "amd64/ubuntu-22.04"
     pool:
-      vmImage: 'ubuntu-20.04'
+      vmImage: 'ubuntu-22.04'
 
     steps:
     - checkout: self
       submodules: true
     - script: |
         sudo apt-get update
-        sudo apt-get install -y make wget libtool m4 autoconf dh-exec libdebhelper-perl=13.6ubuntu1~bpo20.04.1 debhelper=13.6ubuntu1~bpo20.04.1 \
+        sudo apt-get install -y make wget libtool m4 autoconf dh-exec libdebhelper-perl=13.6ubuntu1~bpo22.04.1 debhelper=13.6ubuntu1~bpo22.04.1 \
```

Comment:
Change this to `sudo apt-get install -y make wget libtool m4 autoconf dh-exec debhelper`

Reply:
@saiarcot895 , so, we don't need libdebhelper-perl?